1.1.0-preview1/jessie/build-msbuild/Dockerfile

```diff
@@ -1,10 +1,4 @@
-FROM microsoft/dotnet-nightly:rel-1.0.0-sdk
-
-# Install .NET Core 1.1 - sdk does not currently include this
-ENV DOTNET_SHAREDFX_VERSION 1.1.0-preview1-001100-00
-RUN curl -SL https://dotnetcli.azureedge.net/dotnet/release/1.1.0/Binaries/$DOTNET_SHAREDFX_VERSION/dotnet-debian-x64.$DOTNET_SHAREDFX_VERSION.tar.gz --output sharedfx.tar.gz \
-    && tar -zxf sharedfx.tar.gz -C /usr/share/dotnet \
-    && rm sharedfx.tar.gz
+FROM microsoft/dotnet-nightly:1.1-sdk-msbuild
```

Comment:
Using 1.1 not 1.1.0 was intentional so that we'll get 1.1.x updates when that starts happening.